tools/packaging/packaging.psm1

```diff
@@ -1376,15 +1376,20 @@ function New-ManGzip
 {
     param(
         [switch]
-        $IsPreview
+        $IsPreview,
+
+        [switch]
+        $IsLTS
     )
 
     Write-Log "Creating man gz..."
     # run ronn to convert man page to roff
     $RonnFile = "$RepoRoot/assets/pwsh.1.ronn"
-    if ($IsPreview.IsPresent)
+
+    if ($IsPreview.IsPresent -or $IsLTS.IsPresent)
     {
-        $newRonnFile = $RonnFile -replace 'pwsh', 'pwsh-preview'
+        $prodName = if ($IsPreview) { 'pwsh-preview' } else { 'pwsh-lts' }
```

Comment:
I suggest changing this to `if ($IsLTS) { 'pwsh-lts' } else { 'pwsh-preview' }`.